src/Item.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export default function Item({ restaurantInfo }) {
+  const { name, type, location } = restaurantInfo;
```

Comment:
`List.jsx`에서 props를 destructuring해서 넘겨주면 좋을 것 같아요~

```suggestion
export default function Item({ name, type, location }) {
```

Reply:
이렇게 작성하는 게 더 좋을 것 같네요. 감사합니다.
근데 궁금한게 이렇게 작성할 경우, name, type, location이 어떤 이름인지, 타입인지 모르지 않을 까요?
이에 대해서 어떻게 생각하시나요?

Reply:
맞아요. 제가 잘못 생각했던 것 같습니다.😅 
제가 말씀드린 경우가 더 좋은 케이스는 `restaurantInfo`에 Item에서 쓰는 `name, type, location`외에 다른 정보들이 있는 경우에 유용할 것 같고, 지금과 같은 케이스에서는 `restaurantInfo`를 넘겨주는게 괜찮을 것 같네요~